core/benchmarks/bm-nosonar_node_api.cpp

```diff
@@ -96,6 +97,8 @@ class math_bulk_op : public gr::Block<math_bulk_op<T, op>, gr::PortInNamed<T, "i
     }
 };
 
+// NOTE: math_bulk_op can not be registered to the block registry (GR_REGISTER_BLOCK) as it has NTTPs
```

Comment:
This is one of the prominent use-cases why we use C++: minimising code-duplication through templating.

We must find an acceptable solution for this.

Reply:
As written in the general comment, we could convert this to a type param.